coordinator/shard_mapper_test.go

```diff
@@ -74,7 +75,7 @@ func TestLocalShardMapper(t *testing.T) {
 		t.Fatalf("unexpected number of shard mappings: %d", len(m.ShardMap))
 	}
 
-	if _, err := ic.CreateIterator(measurement, query.IteratorOptions{}); err != nil {
+	if _, err := ic.CreateIterator(context.TODO(), measurement, query.IteratorOptions{}); err != nil {
```

Comment:
Can you modify these various `context.TODO()` calls to what they're supposed to be?

Reply:
I'll change them to `Background`

Reply:
Done